src/main/java/seedu/address/model/AppointmentBook.java

```diff
@@ -142,6 +145,24 @@ public String toString() {
         return appointments.asUnmodifiableObservableList();
     }
 
+    @Override
+    public AppointmentStatistics getAppointmentBookStatistics() {
+        Date td = new Date(LocalDate.now());
```

Comment:
Would it be clearer to change the variable name `td` to something more concrete, maybe `todayDate`?